src/Semaphore/SemaphoreCharacter.tsx

```diff
@@ -9,58 +10,21 @@ type Props = {
 };
 
 type State = {
-  character: Character,
-  left: number,
-  right: number,
+  character: Character
```

Comment:
nit: you can leave the comma, it doesn't hurt anything and saves another diff in the future.

Reply:
Actually, I don't think you need this state at all, just use the props directly in the render method.